app/models/ext_management_system.rb

```diff
@@ -59,7 +59,7 @@ def self.supported_types_and_descriptions_hash
   has_many :customization_specs, :foreign_key => "ems_id", :dependent => :destroy, :inverse_of => :ext_management_system
   has_many :storage_profiles,    :foreign_key => "ems_id", :dependent => :destroy, :inverse_of => :ext_management_system
   has_many :physical_servers,    :foreign_key => "ems_id", :dependent => :destroy, :inverse_of => :ext_management_system
-  has_many :configuration_templates, :foreign_key => "ems_id", :dependent => :destroy, :inverse_of => :ext_management_system
+  has_many :customization_scripts, :foreign_key => "manager_id", :dependent => :destroy, :inverse_of => :ext_management_system
```

Comment:
@bdunne Is it safe to leave this here? Or should it also be moved to the other PR since it uses the renamed "manager_id" field?

Reply:
This is fine.